gp2gp-translator/src/test/java/uk/nhs/adaptors/pss/translator/TestUtility.java

```diff
@@ -28,6 +32,32 @@ public static CV createCv(String code) {
         return createCv(code, "", "");
     }
 
+    public static CD createCd(String code, String codeSystem, String displayName) {
+        final CD cd = new CD();
```

Comment:
Would it be worth implementing the [builder pattern](https://refactoring.guru/design-patterns/builder) for `CD` and `CV` types, seems as if it could be handy.

Reply:
**TODO** another PR.